neqo-transport/src/connection.rs

```diff
@@ -611,25 +603,39 @@ impl Connection {
         }
     }
 
-    /// Set the dest connection ID that was originally chosen by the client.
-    pub(crate) fn set_original_destination_cid(&mut self, odcid: ConnectionId) {
-        assert_eq!(self.role, Role::Server);
-        qtrace!([self], "Called set_original_destination_cid {}", odcid);
-        self.tps
-            .borrow_mut()
-            .local
-            .set_bytes(tparams::ORIGINAL_DESTINATION_CONNECTION_ID, odcid.to_vec());
-        self.remote_original_destination_cid = Some(odcid);
+    /// `odcid` is their original choice for our CID, which we get from the Retry token.
+    /// `remote_cid` is the value from the Source Connection ID field of
+    ///   an incoming packet: what the peer wants us to use now.
+    /// `retry_cid` is what we asked them to use when we sent the Retry.
+    pub(crate) fn set_retry_cids(
```

Comment:
This wraps up several functions from before (those shown on the left).

One thing that this does not do is set `remote_original_destination_cid`.  The server doesn't use it.